docs/howto/install_package.md

```diff
@@ -0,0 +1,103 @@
+# HOWTO: Install package
+
+## Introduction
+A package can be installed using `elastic-package install` command. This command uses the Kibana API to install the package in Kibana.
+
+## Kibana < 8.7.0
+For versions of `Kibana<8.7.0`, the packages must be exposed via the Package Registry.
+
+In case of development, this means that the package should be built previously and then the Elastic stack must be started. Or at least, package-registry service needs to be restarted in the Elastic stack.
+
+```shell
+eval "$(elastic-package stack shellinit)"
+elastic-package build -v
+elastic-package stack up -v -d  # elastic-package stack up -v -d --services package-registry
+elastic-package install -v
+```
+
+## Kibana >= 8.7.0
+Starting with Kibana 8.7.0, there is a new API to install the packages directly without the need to be exposed in Package Registry. It can be use zip files built using `elastic-package build`command.
+
+**NOTE**: These methods assume that the packages have been validated previously.
+
+From Kibana 8.7.0 version, elastic-package install has been updated:
+- a new parameter `--zip` has been added to allow installing a specific zip file.
```

Comment:
I think you can talk in present here, this past tense won't work in a few weeks :slightly_smiling_face: 
```suggestion
From Kibana 8.7.0 version, `elastic-package install` is able to install packages through the upload API.
- The `--zip` parameter can be used to install a specific zip file.
```